terraform/010-cluster/main.tf

```
@@ -16,6 +28,16 @@ module "cloudflare-sg" {
   vpc_id = module.vpc.id
 }
 
+/*
+ * Secondary vpc security group to limit traffic to Cloudflare IPs
+ */
+module "cloudflare-sg_secondary" {
+  count = var.aws_region_secondary == "" ? 0 : 1
+
+  source = "github.com/silinternational/terraform-modules//aws/cloudflare-sg?ref=8.1.0"
+  vpc_id = module.vpc_secondary.id
```

Anything that references a resource or module that has a `count` or `for_each` meta-argument must include an array index. The [`one` function](https://developer.hashicorp.com/terraform/language/functions/one) helps make it a little more concise. I'm only commenting on the first instance of this, since there are numerous places below.
```suggestion
  vpc_id = one(module.vpc_secondary[*].id)
```